roles/quartermaster/tasks/main.yml

```diff
@@ -24,62 +36,49 @@
     path: "{{ item }}"
     state: directory
     mode: 0755
-    owner: quartermaster
-    group: quartermaster
+    owner: "{{ quartermaster_user }}"
+    group: "{{ quartermaster_group }}"
   with_items:
-    - /etc/quartermaster
-    - /var/log/quartermaster
-    - /opt/quartermaster
-
-- name: Download quartermaster binary
-  get_url:
-    url: "{{ quartermaster_install_url }}"
-    dest: /tmp
+    - "{{ quartermaster_home_dir }}"
+    - "{{ quartermaster_log_dir }}"
+    - "{{ quartermaster_venv_dir }}"
+    - "{{ quartermaster_config_dir }}"
 
-- name: Install quartermaster binary
-  unarchive:
-    src: "/tmp/{{ quartermaster_install_zip_filename }}"
-    dest: /opt/quartermaster
-    remote_src: True
+- block:
+    ### This is hacky, but seems necessary to seed a venv w/ py3,
+    #   then use that venv's pip...
+  - name: Seed quartermaster py3 venv
+    pip:
+      name: iso8601
```

Comment:
I was using iso8601 to test locally. I think we can avoid installing iso8601 if we just use pip here instead, so that the pip task is redundantly installing pip and noop'ing since it should be already there.